TCAT/Views/LiveIndicator.swift

```diff
@@ -16,8 +16,10 @@ class LiveIndicator: UIView {
 
     // MARK: State vars
 
-    let lineWidth: CGFloat
-    let size: LiveIndicatorSize
+    private var lineWidth: CGFloat {
```

Comment:
does `size` ever change after we set its initial value in the `init`? if not can we just set `lineWidth` after we set `size`? typically computed properties are only used when its value may change over time